feat(RozierBundle): Rewrote some Rozier legacy javascript into HTML custom-elements

ambroisemaupate opened the pull request.

manuelodelain commented:
IMO this PR is conflicting with https://github.com/roadiz/core-bundle-dev-app/pull/123

If we plan to :

- use Typescript for our codebase,
- build assets and make these compiled files available through this bundle,
- keep the door open to advanced features if needed / possible (e.g. HMR, tree shaking, ...),
- do not plan to use other Symfony UX bundles,

then I think we should stick to Vite, and do not include AssetMapper, or at least do not using it for the front-end assets (JS + CSS)?

ambroisemaupate commented:
> IMO this PR is conflicting with #123
> 
> If we plan to :
> 
>     * use Typescript for our codebase,
> 
>     * build assets and make these compiled files available through this bundle,
> 
>     * keep the door open to advanced features if needed / possible (e.g. HMR, tree shaking, ...),
> 
>     * do not plan to use other Symfony UX bundles,
> 
> 
> then I think we should stick to Vite, and do not include AssetMapper, or at least do not using it for the front-end assets (JS + CSS)?

I've removed symfony/asset-mapper and moved new JS files into the main.js entrypoint.
